scripts/daily-review-prompt.md

```
@@ -226,18 +226,40 @@ gh run list --workflow=flb-shadow-snapshot.yml --limit 5 --json conclusion,creat
+
+Rules:
+- The verdict is keyed on the **TRADEABLE cohort at REAL cost** (the build gate:
+  n ≥ 100, real-cost t ≥ 2, avg > 0). `event_long` is shadow-only — never read it,
+  the pooled row, or any flat-cost number as the verdict.
+- In-sample reference: +2.24%/trade, t=3.49.
+- If the report shows a **flat-vs-real discrepancy ⚠️** (flat t ≥ 2 while real
+  t < 2), surface it prominently — it means the flat column is being mistaken for
+  an edge.
+- If you must run SQL manually, segment by cohort and use `net_pnl_real` on the
+  enterable subset:
+  ```sql
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)